src/main/java/org/opentripplanner/routing/framework/DebugTimingAggregator.java

```diff
@@ -1,237 +1,242 @@
 package org.opentripplanner.routing.framework;
 
+import io.micrometer.core.instrument.Clock;
+import io.micrometer.core.instrument.DistributionSummary;
+import io.micrometer.core.instrument.MeterRegistry;
+import io.micrometer.core.instrument.Metrics;
+import io.micrometer.core.instrument.Timer;
 import java.util.ArrayList;
 import java.util.List;
-import javax.annotation.Nullable;
 import org.opentripplanner.api.resource.DebugOutput;
 import org.opentripplanner.api.resource.TransitTimingOutput;
+import org.opentripplanner.ext.actuator.ActuatorAPI;
+import org.opentripplanner.util.OTPFeature;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 /**
  * Keeps account of timing information within the different parts of the routing process, and is
- * responsible of logging that information.
+ * responsible for logging that information.
  */
 public class DebugTimingAggregator {
```

Comment:
I think this is a good start, but measuring on an individual request is not accurate and may lead to poor statistics. We should probably have a long term goal of replacing this with distribution based timers. I vote for removing the DebugTimingAggregator from the various APIs responses - and only make it available through the Health API. But, not now - one step at the time.

Reply:
We briefly talked about this on todays dev-meeting. Conclusion, we have the new metrics available in the Health API, external tools (profilers) and can log the new metrics - so to ease maintenance we can remove the old performance metrics response. We might do that after the release of 2.1.